src/Tribe/Capabilities.php

```diff
@@ -1,38 +1,106 @@
 <?php
 
 class Tribe__Events__Capabilities {
+
+	/**
+	 * The transient key for delayed capabilities updates.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	protected $key_delayed_capabilities_update = '_tribe_events_delayed_capabilities_update';
+
+	/**
+	 * An array of roles to update capabilities.
+	 *
+	 * @since TBD
+	 *
+	 * @var array
+	 */
+	protected $roles = [
+		'administrator',
+		'editor',
+		'author',
+		'contributor',
+		'subscriber',
+	];
+
+	/**
+	 * An array of capabilities aliases by role.
+	 *
+	 * @since TBD
+	 *
+	 * @var array
+	 */
+	private $cap_aliases = [
+		'editor'      => [ // full permissions to a post type
+		                   'read',
+		                   'read_private_posts',
+		                   'edit_posts',
+		                   'edit_others_posts',
+		                   'edit_private_posts',
+		                   'edit_published_posts',
+		                   'delete_posts',
+		                   'delete_others_posts',
+		                   'delete_private_posts',
+		                   'delete_published_posts',
+		                   'publish_posts',
+		],
+		'author'      => [ // full permissions for content the user created
+		                   'read',
+		                   'edit_posts',
+		                   'edit_published_posts',
+		                   'delete_posts',
+		                   'delete_published_posts',
+		                   'publish_posts',
```

Comment:
* :warning: Found precision alignment of 3 spaces.
* :no_entry_sign: Tabs must be used to indent lines; spaces are not allowed
* :no_entry_sign: Array item not aligned correctly; expected 12 spaces but found 27

<sub>hash: 2818d7e0c0744a8e58c889a84a690555</sub>